src/Raven.Server/Documents/Indexes/Persistence/Lucene/LuceneIndexWriteOperation.cs

```diff
@@ -203,7 +203,7 @@ public override void Delete(LazyStringValue key, IndexingStatsScope stats)
                 _writer.DeleteDocuments(_documentId.CreateTerm(key), _state);
         }
 
-        public override void DeleteTimeSeries(LazyStringValue docId, LazyStringValue key, IndexingStatsScope stats)
+        public override void DeleteByPrefix(LazyStringValue key, IndexingStatsScope stats)
```

Comment:
Below we call `_writer.DeleteTimeSeries(term, _state);` - shouldn't it be renamed to `writer.DeleteByPrefix` as well?

Reply:
done